src/core/types/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod telegram_passport;
```

Comment:
Не хватает перевода строки
```suggestion
pub mod telegram_passport;

```